app/move/controllers/update/document.test.js

```diff
@@ -0,0 +1,267 @@
+const moveService = require('../../../../common/services/move')
+const CreateDocument = require('../create/document')
+
+const UpdateBaseController = require('./base')
+const DocumentController = require('./document')
+
+const MixinProto = CreateDocument.prototype
+
+const controller = new DocumentController({ route: '/' })
+const ownProto = Object.getPrototypeOf(controller)
+
+describe('Move controllers', function() {
+  describe('Update documents controller', function() {
+    it('should extend UpdateBaseController', function() {
+      expect(Object.getPrototypeOf(ownProto)).to.equal(
+        UpdateBaseController.prototype
+      )
+    })
+
+    describe('When mixing in create controller', function() {
+      it('should copy middlewareSetup from CreateDocument', function() {
+        expect(controller.middlewareSetup).to.exist.and.equal(
+          MixinProto.middlewareSetup
+        )
+      })
+
+      it('should copy setNextStep from CreateDocument', function() {
+        expect(controller.setNextStep).to.exist.and.equal(
+          MixinProto.setNextStep
+        )
+      })
+
+      it('should copy saveValues from CreateDocument', function() {
+        expect(controller.saveValues).to.exist.and.equal(MixinProto.saveValues)
+      })
+
+      it('should copy errorHandler from CreateDocument', function() {
+        expect(controller.errorHandler).to.exist.and.equal(
+          MixinProto.errorHandler
+        )
+      })
+
+      it('should only have the expected methods of its own', function() {
+        const ownMethods = ['configure', 'getUpdateValues', 'successHandler']
+        const mixedinMethods = Object.getOwnPropertyNames(MixinProto)
+        const ownProps = Object.getOwnPropertyNames(ownProto).filter(
+          prop => !mixedinMethods.includes(prop) || ownMethods.includes(prop)
+        )
+        expect(ownProps).to.deep.equal(ownMethods)
+      })
+    })
+
+    describe('#configure', function() {
+      const req = {
+        session: {
+          currentLocation: {},
+        },
+      }
+      const res = {}
+      let nextSpy
+      beforeEach(async function() {
+        sinon.stub(MixinProto, 'configure')
+        nextSpy = sinon.spy()
+      })
+
+      context('When current location does allow uploads', function() {
+        beforeEach(function() {
+          req.session.currentLocation.can_upload_documents = true
+          controller.configure(req, res, nextSpy)
+        })
+
+        it('should not call next', function() {
+          expect(nextSpy).to.not.be.called
```

Comment:
I wasn't aware `to.not.be` existed too. I've always used `not.to.be`..._to be_ or _not to be_...

Reply:
I hate mocha

Reply:
Wonder if there's an eslint rule to enforce a particular variant? Manually flipped for now